AppBidding-MoPub/AppBidding-MoPub/AdConfigurations.h

```diff
@@ -0,0 +1,28 @@
+//
+//  AdConfigurations.h
+//  AppBidding-MoPub
+//
+//  Created by Abdoul Guisset.
+//  Copyright © 2020 Abdoul. All rights reserved.
+//
+#import <Foundation/Foundation.h>
+@import CriteoPublisherSdk;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AdConfigurations : NSObject
+
+# pragma mark - MoPub
+
++ (NSString*) mopubBannerAdUnitId;
++ (NSString*) mopubInterstitialAdUnitId;
+
+# pragma mark - Criteo
+
++ (NSString*) criteoPublisherId;
++ (CRBannerAdUnit *)criteoBannerAdUnit;
```

Comment:
Here, you're having a space before the * and none after the ). On the lines above, it is the opposite. For the moment there is no clear guidelines on the code style required for PR. In such case I propose to be consistant within your PR.

Unfortunately, the existing code is not consistant in term of code style, in this case you can mimic the code style on the ios-publisher-sdk: https://github.com/criteo/ios-publisher-sdk/blob/main/CriteoPublisherSdk/Sources/CR_BidManager.h (as describe here: https://github.com/criteo/ios-publisher-sdk/blob/main/DEVELOPMENT.md#coding-style)

Reply:
I will be consistent with this page https://github.com/criteo/ios-publisher-sdk/blob/main/CriteoPublisherSdk/Sources/CR_BidManager.h

I changed the code to:
`+ (NSString *)criteoPublisherId;
+ (CRBannerAdUnit *)criteoBannerAdUnit;`